core/templates/dev/head/pages/collection_editor/CollectionEditorNavbarBreadcrumbDirective.js

```diff
@@ -15,7 +15,6 @@
 /**
  * @fileoverview Controller for the navbar breadcrumb of the collection editor.
  */
-
```

Comment:
Let's not remove the extra line, makes the code more readable!

Reply:
Done. I've added the line.